qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -50,7 +50,7 @@ class TabDeletedError(Exception):
     """Exception raised when _tab_index is called for a deleted tab."""
 
 
-class TabbedBrowser(tabwidget.TabWidget):
```

Comment:
Does this need to be a `QWidget`? I think it simply could be a `QObject` instead?